lua/compe/source.lua

```diff
@@ -221,19 +229,32 @@ Source.documentation = function(self, completed_item)
           context = Context.new({}, {});
           callback = Async.guard('Source.documentation#callback', vim.schedule_wrap(function(document)
             if document and #document ~= 0 then
-              vim.call('compe#documentation#open', document)
+              local pum = vim.fn.pum_getpos()
+              if pum and not vim.tbl_isempty(pum) then
+                local col = pum.col + pum.width + (pum.scrollbar and 1 or 0)
+                local _buf, win = vim.lsp.util.open_floating_preview(document, "markdown", {
```

Comment:
I think `vim.lsp.util.open_floating_preview` would cause flicker (because it does not re-use existing floating-window).

Reply:
And I'm afraid that it is too nightly API...

Reply:
Too nightly is not a problem. We are all using dev version here, expecting things to break. Don't worry about it.

Reply:
But I will receive issues (maybe)...

Reply:
ok, I'll just use the stylize_markdown to render in a custom floating win, so we can reuse the float.

Nightly is already a requirement for compe, so I donit think this is an issue. We're also just a couple of weeks from release now.

Reply:
New version reuses the same window.